openvm/src/lib.rs

```diff
@@ -735,9 +735,9 @@ mod tests {
         assert_eq!(machines.len(), 1);
         let m = &machines[0];
         // TODO we need to find a new block because this one is not executed anymore.
-        assert_eq!(m.width, 157);
+        assert_eq!(m.width, 125);
```

Comment:
This is a regression by 2 columns compared to the pre-#2717 code, see:
https://github.com/powdr-labs/powdr-openvm/pull/128/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R734

I don't yet understand why.

Reply:
Sounds good to me, thanks for the analysis

Reply:
Sounds good